### ISSUE TITLE
Sort out UsersPage waitTillPageIsLoaded() to be reliable

### ISSUE THREAD

**Issue by phil-davis** [owncloud/user_management#167]
See comments in #166 

`UsersPage` `waitTillPageIsLoaded()` has a `sleep(1)` added to the end of it.

If we take that out, then tests that check for notifications will fail quite often:
```
  Scenario Outline: change quota to an invalid value                                                                # /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:38
    When the administrator changes the quota of user "user1" to the invalid string "<wished_quota>" using the webUI # WebUIUsersContext::theAdministratorSetsInvalidQuotaOfUserUsingTheWebUI()
    Then a notification should be displayed on the webUI with the text 'Invalid quota value "<wished_quota>"'       # WebUIGeneralContext::aNotificationShouldBeDisplayedOnTheWebUIWithTheText()
    And the quota of user "user1" should be set to "<wished_quota>" on the webUI                                    # WebUIUsersContext::quotaOfUserShouldBeSetToOnTheWebUI()

    Examples:
      | wished_quota |
      | stupidtext   |
SCENARIO RESULT: (fail)
      | 34,54GB      |
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'Invalid quota value "34,54GB"'
        +''
SCENARIO RESULT: (fail)
      | 30/40GB      |
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'Invalid quota value "30/40GB"'
        +'There were problems with the code integrity check. More information… Invalid quota value "30/40GB"'
SCENARIO RESULT: (fail)
      | 30/40        |
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'Invalid quota value "30/40"'
        +''
SCENARIO RESULT: (pass)
      | 3+56 B       |
Page\UsersPage::setQuotaOfUserTo INFORMATION: notificationId 'notification' did not become visible
SCENARIO RESULT: (fail)
      | -1 B         |
        │ Page\UsersPage::setQuotaOfUserTo INFORMATION: notificationId 'notification' did not become visible
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'Invalid quota value "-1 B"'
        +''

--- Failed scenarios:

    /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:47
    /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:48
    /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:49
    /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:51

15 scenarios (11 passed, 4 failed)
99 steps (91 passed, 4 failed, 4 skipped)
4m2.12s (24.23Mb)
```

There are 3 different outcomes in the above:

1) It does not find any notification
2) It finds an "integrity check" warning - why is that happening???
3) `notification` id did not even become visible

Good luck working out what is going on ;)

**Comment by individual-it** [comment]
the real problem seems to be the integrity check. It fails on master QA build  owncloud/core#34787 see PR #168 
the `sleep(1)` helps because I guess by the time the message about the check is gone

**Comment by individual-it** [comment]
closed in #169

**Comment by individual-it** [comment]
An other level of the issue is that the app itself is cloned from git and not signed. see https://github.com/owncloud/QA/issues/614
